azext_iot/central/params.py

```diff
@@ -159,6 +162,13 @@ def load_deprecated_iotcentral_params(self, _):
             "This enables running cli commands against non public/prod environments",
         )
 
+        with self.argument_context("iotcentral app monitor-events") as context:
```

Comment:
just add module-id to the `iotcentral` subcommand group as a whole (unless thats still causing issues). See how `timeout` is done

Reply:
i.e. unparent this from `iotcentral app monitor-events`